app/frontend/packs/nationalities-component.js

```diff
@@ -0,0 +1,117 @@
+const nationalitiesComponent = () => {
+  const pageHasErrors = document.querySelector(".govuk-error-summary");
+  if (pageHasErrors) return;
+
+  const secondSelect = document.getElementById(
+    "candidate-interface-nationalities-form-other-nationality2-field"
+  );
+
+  const thirdSelect = document.getElementById(
+    "candidate-interface-nationalities-form-other-nationality3-field"
+  );
+
+  const secondFormLabel = document.querySelector(
+    "[for=candidate-interface-nationalities-form-other-nationality2-field]"
+  );
+  const thirdFormLabel = document.querySelector(
+    "[for=candidate-interface-nationalities-form-other-nationality3-field]"
+  );
+
+  let addNationalityButton = null;
+
+  addRemoveLink(secondSelect.id);
+
+  addRemoveLink(thirdSelect.id);
+
+  addAddNationalityButton(
+    "#candidate-interface-nationalities-form-other-other-conditional"
+  );
+
+  hideSection(secondSelect.id);
+
+  hideSection(thirdSelect.id);
+
+  function addRemoveLink(selector) {
+    const labelEl = document.querySelector(`[for=${selector}]`);
```

Comment:
You don't need to run a `querySelector` here anymore now that `secondFormLabel` is available You could just pass it in as an argument where it's called, like `addRemoveLink(secondFormLabel)`, and then:

```suggestion
  function addRemoveLink(labelEl) {
```